tests/security/authz_td_migration/scripts/authz_td_migration.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -u
+set -o pipefail
+
+source "${REPO_ROOT}/content/en/docs/tasks/security/authorization/authz-td-migration/snips.sh"
+
+max_attempts=10
+
+snip_before_you_begin_3
```

Comment:
Do we need to do snip_before_you_begin_2?

Reply:
snip_before_you_begin_2 sets up the namespace which needs to be a blocking call, so it's called as part of the Go program itself.  This task is really slow to execute, so I think it's timing out.  I have a patch that will improve debugging, which I plan to send out shortly, and then I'll confirm that.